thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -706,6 +704,17 @@ class CJoinGroupList
             head = next;
         }
     }
+public:
+    CJoinGroupList() { }
+    ~CJoinGroupList()
+    {
+        removeAll();
+    }
+    void clear()
```

Comment:
The split looks sensible, but this function doesn't seem to be called.

Reply:
I take it back, my search wasn't finding it.

Reply:
called on [new] lines 3048-3049